src/aux-server/aux-web/shared/scene/Game.ts

```diff
@@ -389,14 +442,16 @@ export abstract class Game implements AuxBot3DFinder {
         rotationValue?: Vector2,
         duration?: number
     ) {
+        let isInstant = duration < 0;
+
         this.interaction.addOperation(
             new TweenCameraToOperation(
                 cameraRig,
                 this.interaction,
                 position,
                 zoomValue,
                 rotationValue,
-                duration
+                isInstant
```

Comment:
@shaneYeti Is this a fix for something? I don't see any changes to `TweenCameraToOperation` so I want to make sure this doesn't break anything.

Reply:
changed back